app/views/admin/extensions/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_with(url: toggle_admin_extensions_path, method: :post, local: true) do |f| %>
```

Comment:
Is it deliberate that the 'Extensions' option doesn't appear in the admin navigation menu (i.e. you have to know the URL to go to)? I'm assuming it is, but thought I best ask just in case!

![admin navigation menu](https://user-images.githubusercontent.com/20773163/158799878-0a18c1f4-3042-40eb-94dd-d35c433f54e7.png)

Reply:
Yes extensions is a third level navigation item: 

<img width="1680" alt="Screenshot 2022-03-17 at 13 47 22" src="https://user-images.githubusercontent.com/2786819/158802209-8d3ba6f3-8129-4f89-9d3a-abbbc455c6a5.png">


Aside note: The navbar seems to have regressed so its not highlighting "Developer Tools", and I noticed it a while ago, you can read more about it [here](https://github.com/forem/forem/pull/16888#discussion_r829029987). Either myself or Jeremy will tackle it in a separate PR.

Reply:
> The navbar seems to have regressed so its not highlighting "Developer Tools"

ah yes, I think that's what threw me! thanks for clarifying!